doc/api/GVT-g_api.rst

```diff
@@ -0,0 +1,114 @@
+.. _GVT-g_api:
+
+===============
```

Comment:
Take a look at our ACRN documentation guidelines at https://projectacrn.github.io/latest/developer-guides/doc_guidelines.html for our project conventions for writing reST files.

For the document title we just use a ``#####`` underline, so change the top of the doc to:

    .. _GVT-g_api:

    ACRN GVT-g APIs
    ###############

Reply:
Thanks.  Will convert to ACRN style instead of kernel-doc style.